cli/cmd/proxy/read/config.go

```diff
@@ -77,20 +77,36 @@ func FetchConfig(ctx context.Context, portForward common.PortForwarder) (*EnvoyC
 	}
 	defer portForward.Close()
 
+	// Fetch the config dump
 	response, err := http.Get(fmt.Sprintf("http://%s/config_dump?include_eds", endpoint))
 	if err != nil {
 		return nil, err
 	}
-	raw, err := io.ReadAll(response.Body)
+	configDump, err := io.ReadAll(response.Body)
 	if err != nil {
 		return nil, err
 	}
 	if err := response.Body.Close(); err != nil {
 		return nil, err
 	}
 
+	// Fetch the clusters mapping
+	response, err = http.Get(fmt.Sprintf("http://%s/clusters?format=json", endpoint))
```

Comment:
No :( unfortunately that’s the origin of this bug. I thought I could get that info by passing include_eds, but that doesn’t actually populate the endpoints.

Reply:
I see that it populates the endpoint config, but the endpoint we need doesn't have cluster name, so we can't find it. I'm not sure why that is though.

Reply:
Nor am I, the `include_eds` doesn't really work like I would have expected. This solution works though. It doesn't add much overhead and there is no way to get this information otherwise AFAIK.